src/collective/fingerpointing/tests/test_utils.py

```diff
@@ -13,15 +12,98 @@ def setUp(self):
         self.request = self.layer['request']
 
     def test_get_request_information(self):
+        from collective.fingerpointing import utils
         self.assertEqual(
-            get_request_information(), ('test_user_1_', 'None'))
+            utils.get_request_information(), ('test_user_1_', 'None')
+        )
 
     def test_get_request_information_anonymous(self):
         from plone.app.testing import logout
         logout()
-        self.assertEqual(get_request_information(), ('-', 'None'))
+        from collective.fingerpointing import utils
+        self.assertEqual(utils.get_request_information(), ('-', 'None'))
 
     def test_get_request_information_cloudflare(self):
         self.request.environ['HTTP_CF_CONNECTING_IP'] = '192.168.1.1'
+        from collective.fingerpointing import utils
         self.assertEqual(
-            get_request_information(), ('test_user_1_', '192.168.1.1'))
+            utils.get_request_information(), ('test_user_1_', '192.168.1.1')
+        )
+
+    def test_audit_log_files(self):
+        from collective.fingerpointing import utils
+        audit_log_files = utils.audit_log_files()
+        # we have at least one log file to process
+        self.assertGreaterEqual(len(audit_log_files), 1)
+        # the first file is the current audit log
+        self.assertEqual('/audit.log', audit_log_files[0][-10:])
+
+
+class UtilsOneLargeLogFileTestCase(unittest.TestCase):
+
+    layer = INTEGRATION_TESTING
+
+    def setUp(self):
+        # fresh audit log
+        from collective.fingerpointing.config import fingerpointing_config
```

Comment:
same here: what's the point on importing them all over the place?

Reply:
see above, keep it please.